### PR TITLE
Updated to LESS 2.7.1

### PR DESCRIPTION
**Pull request by johanberonius**
Quick fix to make it work with LESS `2.7.1`.

### CODE REVIEW

File: lib/lessfile.js [hunk 1]
```diff
@@ -26,20 +19,21 @@ LessFile.prototype = {
   },
 
   parse: function(callback) {
-    var self = this, parser = this._getParser();
-    parser.parse(fs.readFileSync(this.file, 'utf-8'), function(err) {
-      self.imports = Object.keys(parser.imports.files);
+    var self = this;
+    less.render(fs.readFileSync(this.file, 'utf-8'), {
```
Comment (jgonera) [inline lib/lessfile.js]: Let's put each argument to this method in a new line for clarity.

File: lib/lessfile.js [hunk 2]
```diff
@@ -26,20 +19,21 @@ LessFile.prototype = {
...
+        paths: path.dirname(this.file),
+        filename: this.file
+      }, function(err, output) {
+      self.imports = output.imports;
```
Comment (jgonera) [inline lib/lessfile.js]: Indentation.

File: lib/lessfile.js [hunk 3]
```diff
@@ -63,14 +57,16 @@ LessFile.prototype = {
 
   compile: function(compileOptions, callback) {
     var self = this;
-    this._getParser().parse(fs.readFileSync(this.file, 'utf-8'), function(err, tree) {
+    compileOptions.paths = path.dirname(this.file);
```
Comment (jgonera) [inline lib/lessfile.js]: Can we DRY this (L24-25)?

**Review by jgonera — CHANGES_REQUESTED** [review]
Thanks for the patch! A few minor comments inline.

**Review by johanberonius — COMMENTED** [review]
Parse and render methods now updated to use the same options, just like`less.Parser` used before.

**Comment by nutgaard** [comment]
@jgonera Anything missing from getting this merged and published?

**Comment by jgonera** [comment]
Sorry, I'm not going to maintain this project anymore since modern web development uses Webpack/Rollup for everything. I'm closing this PR before archiving the repo, feel free to fork it if you really need autoless.

**Comment by jgonera** [comment]
Sorry for not merging/reviewing again back in the day.

**Pull request closed without merging**